app/assets/stylesheets/govuk_publishing_components/components/_layout-super-navigation-header.scss

```diff
@@ -609,7 +609,9 @@ $button-pipe-colour: darken(govuk-colour("mid-grey"), 20%);
   }
 
   &:after {
-    @include pseudo-underline;
+    @include govuk-media-query($from: "desktop") {
+      @include pseudo-underline;
+    }
```

Comment:
I think you can maybe remove this block altogether since (I think...) the button is only seen on larger screens when the user is zoomed in to more than 200% but then the screen size is too small for the media query to have any effect.

Reply:
tested this and yeah that is the case, will remove